function/toshandler.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const validation = require('./validation.js');
```

Comment:
Can you destructure this?

Reply:
I certainly could ... 
```
const {validateRequestUrl, validateRequestMethod, validateContentType,
    requireAuthorizationHeader, validateInputs} = require('./validation.js');
```
felt more verbose to me. Is there a shorthand for that?

Reply:
No, but it saves you calling them each as `validation.foo` every time.